src/storage/mvcc/meta.rs

```diff
@@ -69,11 +74,37 @@ impl Meta {
     pub fn push_item(&mut self, item: MetaItem) {
         self.pb.mut_items().insert(0, item);
     }
+
+    pub fn next_index(&self) -> Option<u64> {
+        match self.pb.get_next() {
+            0 => None,
```

Comment:
@siddontang I think 0 is better than FIRST_META_INDEX, 0 represents 'no more metas' here, kind like nil.